### DISCUSSION TITLE
Can I run a mutation from my remote schema on Postgres delete?

### DISCUSSION THREAD

**Discussion by giorgosera** [hasura/graphql-engine#9304] [Question]
Hey everyone! Here's my use case: _**I have a 3rd party service that I need to notify once an object in my db is deleted.**_

So what i want to do is:

- Run the delete mutation via Hasura's autogenerated API
- Upon deletion I want to call a custom mutation at a remote schema

What I've considered: 

- Using events. But it seems to me that the webhook endpoint can only be REST. I cannot trigger a Graphql mutation at my remote schema. Please correct me if I'm wrong
- Overriding the autogenerated Hasura delete mutation by re-implementing it in my remote schema and notify the 3rd party with custo code there. But this means that I'm losing the benefit of having this mutation generated for me.

Is there anything else that I might  consider here?

Thank you 🙏

**Comment by adas98012** [comment]
@giorgosera 
You can use RESTified GraphQL endpoint (on Remote Schema), fronting your GraphQL mutation:

https://hasura.io/docs/latest/api-reference/restified/

**Reply by giorgosera** [reply]
Thanks for the reply!
